tests/e2e/cmp_test.go

```diff
@@ -49,6 +49,11 @@ var _ = Describe("odoCmpE2e", func() {
 
 	Context("odo component creation", func() {
 
+		// This is required as some CI's (such as TravisCI) requires a little bit more time
+		It("should set an appropriate test-configured PodTimeout", func() {
+			runCmd("odo utils config set PodTimeout 120")
```

Comment:
But 120 is the default value right?

Reply:
You're right. I'll change it to a different value. 

The reason I set it to "120" again is because I previously set it to 10 and it was causing timeouts / issues..